composer.json

```diff
@@ -38,6 +38,7 @@
         "coduo/php-matcher": "^2.1",
         "friends-of-behat/context-service-extension": "^1.0",
         "friends-of-behat/cross-container-extension": "^1.0",
+        "friends-of-behat/exclude-specifications-extension": "dev-fix-excluding-feature-files",
```

Comment:
To be changed after merging https://github.com/FriendsOfBehat/ExcludeSpecificationsExtension/pull/22

Reply:
v0.1.1 released :)